CesiumGS/cesium
# Throttle total resources used for environment maps

# Description

[As demonstrated in this Sandcastle example](https://sandcastle.cesium.com/#c=vVXdc9MwDP9XdH3KWOYk++CjdDtgwMEdO8YKvFAevERtvDp2z3b2wa7/O7LTbEkoPLKHTJZ+kn6SZTXXyjq4FniDBo5B4Q2cohV1xb4HXTQb5eF8qpXjQqGZjWK4nykAoeb6jb4dw5xLi7FXWZSYO6HVR1WInDttetaSF/rGjsGZutXoWhavlai4wwf9euflTM1UQ4rZHBWyAi/rxbTUN+8Nr9Ceo5lirlVBnL1XcJjXKiSH3CDFO9MFyqg2MoYSxaJ0MfBK18qRT7bT1LDJgcoJJ9Ayg5W+xtdSRp4DwFwbiCQ6EOSUvqR/k00Qknd3d+7Bo/zfA/KqQV51kFc9ZA+9bNDLDnq5QXfxebimlbYiFHjcXtIpN44krg7Y3OjqLS4Moo26rgB72f4BS58dHj7NXsAu1fAEUpamaRb3cYeHLD1KD56lTwl19TdU00tCLAmRpT1r07Q+6RJ5IdTikfMZdyVz+oLUXNkoOzja5rYSLi9Dc/60GS3lX0zlajDGH5r05z7cBflFGz5xkyEOwbYR0EbQWPB+v78aokyXV1lWDgJ/qWnmjCJ41N5T7Ok0wzyMHibujgIPJ5AXRXTf77iiiR+DH+S+vk0zfpAGgE4J4+5hAKv8QxnDICtQRrEtLSQJVILebF2di1uUU/GL6GX7z4c4qPitR01zLgmxT8OUDjDr3nndv4dNa5zh+RKLd23HmtZ1kOtWDAJ91r7jm1tc+YItuf0g46ZEh7duDLPRNyWFA9phsAcZ7bXGSvCwxmh1tQsl2nnsTne5zEaMJVNerSS+5Y4noZM2oYgh9IPAFtLN/eLM6EnRt62zKT98vfAPekeQ15dY/EeSR1tI/rOB6X+nmKVbG/kzvLgpV0XOrZPo39RXreUlN2eo6mgzE953FI8m1t1JPGlpvRLVShvn5z4iUg6JFPG0yWVNU+hYbm2bdJJ0XSeFuAZRHG/5wYRccmvJMq9leC6z0ckkIfwfrlKHpfL5Go3kdx5WZiefGiVjbJLQcbunawocRP4N), it's possible for the environment map generation to puts a large load on browser resources when many models are added in the same frame, and that can cause the browser to hang and potentially crash.

This fix ensure that the environment map computations are distributed across several frames once they hit an upper limit, preventing the large simultaneous load on browser resources. 

## Issue number and link

N/A

## Testing plan

1. Try the above Sandcastle at each various option. Even at 10 x 10 grids, frame rate should still be interactive.
1. Make sure that otherwise, environment maps are being populated, such as in the "glTF PBR Extensions" example

# Author checklist
